packages/angular/cli/src/utilities/completion.ts

```diff
@@ -116,6 +116,11 @@ async function shouldPromptForAutocompletionSetup(
     return forceAutocomplete;
   }
 
+  // Don't prompt on Windows because it doesn't support autocompletion.
+  if (process.platform === 'win32') {
+    return false;
+  }
+
```

Comment:
Nit: probably we should use the same logic as yargs to determine if the shell is bash or zsh.

https://github.com/yargs/yargs/blob/db354232705623bbcd8fad362f6a4d6d59650be5/lib/completion.ts#L44-L47